src/views/CoCurricularTranscript/index.js

```diff
@@ -41,6 +42,8 @@ export default class Transcript extends Component {
       /* Retrieve data from server */
       const profile = await user.getProfileInfo();
 
+      this.state.profile = profile;
```

Comment:
This is bad practice, and will break if CoCurricularTranscript ever re-renders. You need to instead use setState:
```suggestion
      this.setState({ profile });
```

Reply:
done